recipes/scikit-odes/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "scikits-odes" %}
+{% set pypiname = "scikits.odes" %}
+{% set version = "2.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz
+  sha256: cb5831a26d6a1325d24fc0800e2232195fe09c369d6e754d2f43276882156e4d
+
+build:
+  number: 0
+  # NOTE : There is a pyproject.yml file that seems to force wheels to be built
+  # so remove it.
+  script:
+    - "/bin/rm -f pyproject.toml"
+    - "{{ PYTHON }} -m pip install . -vv"
+  # NOTE : Will probably need to disable windows.
+  #skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - cython
+    - numpy
+    - pip
+    - python
+    - setuptools
+    - sundials >=5.1.0,<6 # scikits-odes docs say ==5.1.0, hope this is reasonable
+  run:
+    - python
+    - scipy
+    - sundials >=5.1.0,<6
+    - {{ pin_compatible('numpy') }}
```

Comment:
The numpy api is used via Cython for wrapping Sundials so I think this is needed.

Reply:
Yes, it is.